sdk/modelsrepository/azure-iot-modelsrepository/azure/iot/modelsrepository/_client.py

```diff
@@ -45,25 +45,25 @@ class ModelsRepositoryClient(object):
 
     def __init__(self, **kwargs):  # pylint: disable=missing-client-constructor-parameter-credential
         # type: (Any) -> None
-        """
+        """For additional request configuration options, please see [core options](https://aka.ms/azsdk/python/options).
```

Comment:
I feel like the first line of the docstring should be something more like: Client for the Models Repository service (or something like that - maybe there's a line from the README we can use.....)

Reply:
good point, let me fix that